src/goto-symex/symex_target_equation.cpp

```diff
@@ -328,19 +328,20 @@ void symex_target_equationt::constraint(
 }
 
 void symex_target_equationt::convert(
-  prop_convt &prop_conv)
+  prop_convt &prop_conv,
+  message_handlert &message_handler)
 {
   try
   {
-    convert_guards(prop_conv);
-    convert_assignments(prop_conv);
+    convert_guards(prop_conv, message_handler);
```

Comment:
Can you add a `messaget log` member to reduce the churn?

Reply:
I could do, but then the constructor would need to take a `message_handlert` (as constructing a `messaget` without a `message_handlert` is deprecated). Does that seem reasonable?

Reply:
@peterschrammel I have done as you suggested - I think it's thanks to your goto-checker refactoring that this was a fairly smooth sailing.